src/applications/personalization/profile/components/alerts/bad-address/analytics.js

```diff
@@ -0,0 +1,9 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+const recordBadAddressEvent = ({ location = 'unknown' }) => {
+  recordEvent({
+    event: `profile-bad-address-${location}-alert-shown-`,
+  });
+};
+
+export { recordBadAddressEvent };
```

Comment:
This all looks good so far! 

Might want to consider adding the possibility of recording the different actions with this method. Maybe passing an action of `show` vs `click` for since I noticed you will have to record both those events to analytics.

Also was curious as to why there is a dangling `-` at the end of the event name?